.nvmrc

```diff
@@ -1 +1 @@
-18
+20
```

Comment:
updating this to 20 because using 18 created lots of yarn lock differences

it shows that we have been using v20 already